tests/support/expectations.js

```diff
@@ -10,7 +10,6 @@ module.exports = function (request, config, isServer) {
     currentLog = log;
   };
   var superagentPackage = require('superagent/package.json');
-  var superagentUserAgentHeader = isServer ? {'User-Agent': 'node-superagent/' + superagentPackage.version} : {};
```

Comment:
Why does this have been removed ?

Reply:
The user agent is no longer set by the superagent library, so there's no point in testing its presence on the superagent-mock side